towhee/dag/variable_repr.py

```diff
@@ -21,17 +20,16 @@ class VariableRepr(BaseRepr):
 
     Args:
         name:
-            Variable name.
+            The representation name.
         vtype:
             This can be one of many possible variable types, such as a numpy array or
             PyTorch tensor.
         dtype:
             A string or instance of `numpy.dtype` indicating the internal data type for
             this variable.
     """
-
-    def __init__(self, name: str, vtype: str, dtype: str):
-        super().__init__(name)
+    def __init__(self, vtype: str, dtype: str):
+        super().__init__()
         self._vtype = str(vtype)
```

Comment:
Input is already str type, why use str(...)

Reply:
fixed